adagucserverEC/CImageDataWriter.cpp

```diff
@@ -1105,7 +1105,23 @@ int CImageDataWriter::getFeatureInfo(std::vector<CDataSource *>dataSources,int d
               }
             }
             
-            CDBDebug("closestIndex: %d", closestIndex);
+            /* Now we have detected the closest point, check if it is in a certain distance in the map in pixel coordinates */
+            double pointPX = dataSource->getDataObject(o)->points[closestIndex].lon, pointPY = dataSource->getDataObject(o)->points[closestIndex].lat;
+            double gfiPX = getFeatureInfoResult->lon_coordinate, gfiPY = getFeatureInfoResult->lat_coordinate;
+            CImageWarper warper;
+            int status = warper.initreproj(LATLONPROJECTION,srvParam->Geo,&srvParam->cfg->Projection);
```

Comment:
Waarom eigenlijk in LATLONPROJ? Hoe werkt het als we naar de data in een andere projectie kijken?

Reply:
getFeatureInfoResult->lon_coordinate en getFeatureInfoResult->lat_coordinate zijn altijd in lat/lon, dit wordt door CImageDataWriter geregeld.